src/modules/campaign/sagas.ts

```diff
@@ -92,8 +97,10 @@ export function* campaignSagas(
       }) as LocalizedField<SysLink<'Entry'>> | undefined
 
       const campaignFields = campaignField?.[ContentfulLocale.enUS].sys.id
-        ? (entries[campaignField?.[ContentfulLocale.enUS].sys.id]
-            ?.fields as CampaignFields)
+        ? (Object.values(entries).find(
```

Comment:
Shouldn't the entries be a record of <`entry.sys.id`, entry>? If that's the case, there's no need to go through all of the entries, right?

Reply:
good catch